src/en/controllers.md

```diff
@@ -1,40 +1,150 @@
 Title: Juju Controllers
-TODO: Review again soon (created: March 2016)
+TODO:  Figure out the 'block command' situation (including CLI help
+         text). See 'Restricting command usage' section. The old page lives on.
+       Need examples for each command.
+       Figure out how to present those examples due to the foldout bug not
+         allowing syntax highlighting. So far, under 'Creating a controller'
+         there is a link to the old creating page.
 
```

Comment:
The 'workaround' to get examples styled as such (no highlighting, but the correct style) is simply to indent them by 6 spaces or more, in addition to the 3 spaces everything is already indented by